Translations/zh-Hans.lproj/main.strings

```diff
@@ -90,7 +90,7 @@
 "maF-B5-Hb6.title" = "关于 Keka";
 
 /* Class = "NSMenuItem"; title = "Revert to Saved"; ObjectID = "nlc-UW-DRW"; */
-"nlc-UW-DRW.title" = "复原到已存储的版本";
+"nlc-UW-DRW.title" = "还原至已保存";
```

Comment:
Is this and the next below used in Big Sur?

Reply:
I not found it in Big Sur...just fixed of old translation.

Reply:
True, It is no present, neither in Catalina